simple_network_sim/network_of_populations.py

```diff
@@ -104,6 +147,14 @@ def getTotalInAge(nodeState, ageTest):
 
 # CurrentlyInUse
 def getTotalInfectious(nodeState):
+    """Get the total number of infected individuals regardless of age.
```

Comment:
It should be "infectious" and not "infected". There are people who are infected, but not infectious (eg. exposed and hospitalized).

Also, it's worth mentioning this function does that for a specific node.